uber/templates/registration/new_base.html

```diff
@@ -74,6 +74,79 @@
         }
     };
     $(showOrHideWhatText);
+
+    var refreshRow = function(currentForm) {
+        currentForm.closest('tr').find(':contains("Check In")').removeAttr('disabled');
+        currentForm.closest('td').text('paid');
+    };
+
+    var makeStripeAjax = function() {
+        if($("form[action='manual_reg_charge']").length) {
+            $("form[action='manual_reg_charge']").each(function(index) {
+                $(this).get(0).submit = function() {
+                    var data = $(this).serialize();
+                    var currentForm = $(this);
+                    // process data and send ajax request
+
+                    $.ajax({
+                        method: 'POST',
+                        url: '../registration/manual_reg_charge',
+                        dataType: 'json',
+                        data: data,
+                        success: function (json) {
+                            toastr.clear();
+                            var message = json.message;
+                            if (json.success) {
+                                toastr.info(message);
+                                refreshRow(currentForm);
+                            } else {
+                                toastr.error(message);
+                            }
+                        },
+                        error: function () {
+                            toastr.error('Unable to connect to server, please try again.');
+                        }
+                    });
+
+                    // Prevent form submit.
+                    return false;
+                }
+            });
+        }
+    };
+    $(makeStripeAjax);
+    $(function() {
+        $("form[action='mark_as_paid']").each(function(index) {
+            $(this).submit(function (e) {
+                // Prevent form submit.
+                e.preventDefault();
+
+                var data = $(this).serialize();
+                var currentForm = $(this);
+
+                $.ajax({
```

Comment:
These ajax functions take an awful lot of space for what's basically the same function with a different form and URL. I didn't have any good ideas for compressing them, though.